### PR TITLE
Improve lighting

### PR DESCRIPTION
**Pull request by trungleduc**
## Before 
before

## After

after

### CODE REVIEW

**Comment by trungleduc** [comment]
Hey please update snapshots

File: packages/jupytercad-extension/src/mainview.tsx [hunk 1]
```diff
@@ -236,9 +236,9 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene = new THREE.Scene();
 
-      this._scene.add(new THREE.AmbientLight(0xffffff, 0.8)); // soft white light
+      this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      const light = new THREE.SpotLight(0xffffff, 0.2);
+      const light = new THREE.PointLight(0xffffff, 1);
       light.castShadow = true;
```
Comment (martinRenou) [inline packages/jupytercad-extension/src/mainview.tsx:242]: I think we should disable the shadowmap computing. We're not using it so we could save some perf here.

File: packages/jupytercad-extension/src/mainview.tsx [hunk 2]
```diff
@@ -236,9 +236,9 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene = new THREE.Scene();
 
-      this._scene.add(new THREE.AmbientLight(0xffffff, 0.8)); // soft white light
+      this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      const light = new THREE.SpotLight(0xffffff, 0.2);
+      const light = new THREE.PointLight(0xffffff, 1);
```
Comment (martinRenou) [inline packages/jupytercad-extension/src/mainview.tsx:241]: I did not know adding a point light to the camera would make the point light stick to the camera position! That's good to know!

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**